### ISSUE TITLE
[GraphQL]DeleteRequisitionListOutput should not return the deleted Req.list data

### ISSUE THREAD

**Issue by magento-engcom-team** [magento/magento2#31305]
Create a customer and create couple requisition lists for the customer.

```php
mutation {
  deleteRequisitionList(
    requisitionListUid:"{$requisitionListId}"
) {
    status
    requisition_list {
        uid
        name
        description
        }
    }
}
```

Expected Result:
The deleteRequisitionListOutput should return the status and/or lists of un-deleted requisition lists.( to align with deletewishlist) Or just return the status

Actual Result:
The output returns the details of the deleted requisition list along with the status

**Comment by magento-engcom-team** [comment]
The issue was exported from the internal JIRA. The link to the original JIRA issue: https://jira.corp.magento.com/browse/MC-39894

**Comment by magento-engcom-team** [comment]
Hi @magento-engcom-team, @shikhamis11. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-39894` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.
